TipCatDotNet.Api/Data/Models/Analitics/AccountResume.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace TipCatDotNet.Api.Data.Analitics;
+
+public class AccountResume
```

Comment:
I'm not happy with the name, but can't advice anything better. Probably we'll rename it later.